src/deprecated.jl

```diff
@@ -22,3 +22,6 @@ import Base: keys, values, insert!
 
 @deprecate stackdf stackdt
 @deprecate meltdf meltdt
+
+@deprecate readtable CSV.read
```

Comment:
Since DataTables is a new package, I wouldn't bother too much about writing a proper deprecation rule. Just printing an error saying to use CSV.jl could be enough. If you really feel like it, you could pass the arguments which have an equivalent, and print a warning for any argument which cannot be translated.